### ISSUE TITLE
Multiple controls selection error

### ISSUE THREAD

**Issue by mtszkarbowiak** [FlaxEngine/FlaxEngine#2524]
**Issue description**

Editor issues when selecting multiple actors.

**Steps to reproduce**

1. Create control A with canvas as parent.
2. Create controls A1, A2 and A3 with A as parent.
4. Create control B with canvas as parent. (B must be under/behind A)
5. Select A2 A3 and B (in this exact order!).

Editor is spammed with errors.

```
[ 00:10:20.135 ]: [Warning] Failed to setup values and UI for item Brush
[ 00:10:20.136 ]: [Warning] Object does not match target type.
[ 00:10:20.136 ]: [Warning]    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.RuntimePropertyInfo.GetValue(Object obj, Object[] index)
   at FlaxEditor.Scripting.ScriptMemberInfo.GetValue(Object obj) in 
   ...\FlaxEngine\Source\Editor\Scripting\ScriptType.cs:line 661
   at FlaxEditor.CustomEditors.ValueContainer..ctor(ScriptMemberInfo info, ValueContainer instanceValues) in ...\FlaxEngine\Source\Editor\CustomEditors\Values\ValueContainer.cs:line 261
   at FlaxEditor.CustomEditors.Editors.GenericEditor.ItemInfo.GetValues(ValueContainer instanceValues) in ...\FlaxEngine\Source\Editor\CustomEditors\Editors\GenericEditor.cs:line 149
   at FlaxEditor.CustomEditors.Editors.GenericEditor.Initialize(LayoutElementsContainer layout) in ...\FlaxEngine\Source\Editor\CustomEditors\Editors\GenericEditor.cs:line 772

```

Note: Selecting B as first does not trigger the bug. If selection starts from the first control, the bug does trigger neither.

**Flax version**

1.7.2

**Comment by Tryibion** [comment]
I believe that this has already been fixed in 1.8.1

**Comment by mafiesto4** [comment]
It looks like it's working fine now. We've got some improvements to selection some time ago.
